data-raw/chirps_monthly_afg_adm1_historical.R

```diff
@@ -0,0 +1,65 @@
+#' `chirps_monthly_afg_adm1_historical.csv`
+#' `Date:` 2024-04-22
+#' This script generates the chirps_monthly_afg_adm1_historical.csv
+#' 
+#' **Description:**
+#' Using GEE temporally aggregate CHIRPS daily data to monthly.
+#' We then run zonal means  for each yr_mo combination at the admin-1 level 
+#' For all of Afghanistan.
+#' 
+#' **Tip:* 
+#' Script can be run in the background (on Mac with terminal call)
+#' `caffeinate -i -s Rscript data-raw/chirps_monthly_afg_adm1.R`
+
+library(rgee)
+library(tidyrgee)
+library(rhdx)
+library(tidyverse)
+ee_Initialize()
+
+overwrite_csv <-  c(T,F)[2]
+
+chirps_daily_url <- "UCSB-CHG/CHIRPS/DAILY"
+chirps_ic <- ee$ImageCollection(chirps_daily_url)
+
+adm1_fc <- ee$FeatureCollection("FAO/GAUL_SIMPLIFIED_500m/2015/level1")
+
+# filter adm1 to get only those in Afghanistan
+adm1_afghanistan <- adm1_fc$filter(ee$Filter$eq("ADM0_NAME", "Afghanistan"))
+
+# Add to map to quickly check - only for interactive viewing (not backaground job)
+# Map$addLayer(adm1_afghanistan, list(color = "red"), "Afghanistan")
+
+# convert to tidyee IC for easy temporal aggregation
+chirps_tic <-  as_tidyee(chirps_ic)
+
+chirps_monthly_tic <- chirps_tic |> 
+  group_by(year, month) |> 
+  summarise(
+    stat= "sum"
+  )
+
+
+# zonal means for each year-month at admin 1
+df_chirps_monthly <- ee_extract_tidy(
```

Comment:
I keep getting `Error in value[[3L]](cond) : 
  The ee$FeatureCollection (y) must be not higher than 10 000.`, even though the feature collection should just be 34 long. Could it be related to [your issue here](https://github.com/r-spatial/rgee/issues/259)?

Reply:
yeah seems related, but oddly this is not causing issues for me. 
Probably it's calculating `length(y)` as `34 (provinces) *12 (months) * 43 (years)`.

 Did you download the dev version of `{tidyrgee}` (devtools::install_github("r-tidy-remote-sensing/tidyrgee")).  Maybe I implemented a work-around for the issue there, but I thought it would be on CRAN anyway